code/modules/mob/living/silicon/ai/life.dm

```diff
@@ -168,8 +166,6 @@
 									show_laws()
 							sleep(50)
```

Comment:
By the way in the case that this occurs it delays the entire mob dowork that goes through all the lifes